instrumentation/httpclient5/src/main/java/brave/httpclient5/AsyncHandleSendHandler.java

```diff
@@ -0,0 +1,60 @@
+package brave.httpclient5;
+
+import brave.Span;
+import brave.http.HttpClientHandler;
+import brave.http.HttpClientRequest;
+import brave.http.HttpClientResponse;
+import brave.http.HttpTracing;
+import brave.internal.Nullable;
+import brave.propagation.TraceContext;
+import java.io.IOException;
+import org.apache.hc.client5.http.async.AsyncExecCallback;
+import org.apache.hc.client5.http.async.AsyncExecChain;
+import org.apache.hc.client5.http.async.AsyncExecChainHandler;
+import org.apache.hc.client5.http.protocol.HttpClientContext;
+import org.apache.hc.core5.http.HttpException;
+import org.apache.hc.core5.http.HttpRequest;
+import org.apache.hc.core5.http.nio.AsyncEntityProducer;
+
+import static brave.httpclient5.HttpClientUtils.parseTargetAddress;
+
+class AsyncHandleSendHandler implements AsyncExecChainHandler {
+  private final HttpClientHandler<HttpClientRequest, HttpClientResponse> handler;
+  @Nullable
+  private final String serverName;
+
+  AsyncHandleSendHandler(HttpTracing httpTracing) {
+    this.serverName = "".equals(httpTracing.serverName()) ? null : httpTracing.serverName();
```

Comment:
this seems we should fix httpTracing.serverName() to not permit empty string.. mind moving it there? (ex builder coerce empty to null)

Reply:
copy from httpclient(4.x) one, but looks like HttpClientHandler handles this well. Maybe I can add another PR to change the httpclient(4.x) later.